app/app.py

```diff
@@ -60,32 +60,58 @@ def admin():
 
 @app.route("/")
 def choose():
-    return render_template("choose.html", locale="es")
+
+    return render_template("choose.html", products=get_products())
+
+
+def get_products(include_archived=False):
+    """Get all products exluding archivedn by default"""
+    products_path = Path(SHARED_MOUNT_POINT, "products")
+    product_files = list(
+        filter(lambda y: y.is_file(), products_path.iterdir())
+    )  # noqa: E501
+    products = []
+    for path in product_files:
+        with open(path) as fp:
+            product = json.loads(fp.read())
+            if product["active"] == "1":
+                products.append(product)
+    return products
 
 
 @app.route("/request-date-time", methods=["GET", "POST"])
 def set_date_time():
-    return render_template("request-date-time.html")
+    product_id = request.form.get("product_id")
+    print(product_id)
+    return render_template("request-date-time.html",product_id=product_id)
 
 
 @app.route("/deposit", methods=["GET", "POST"])
 def deposit():
-    return render_template("deposit.html")
+    product_id = request.args.get("product")
+    product = get_product(product_id)
+    return render_template("deposit.html", product=product)
 
 
 @app.route("/create-checkout-session", methods=["GET", "POST"])
 def create_checkout_session():
     stripe.api_key = STRIPE_API_KEY
 
-    requested_product = request.form.get("product")
+
+    product_id = request.form.get("product")
     requested_time = request.form.get("time")
     requested_date = request.form.get("date")
     customer_email = request.form.get("email", None)
     customer_name = request.form.get("name", None)
     customer_mobile = request.form.get("mobile", None)
+    product = get_product(product_id)
+    deposit_amount = product["deposit_amount"]
+    product_name = product["product_name"]
 
     metadata = {
-        "requested_product": requested_product,
```

Comment:
Todo: Document the naming convention.

The naming convention must be preserved here- `requested_product` has a significant meaning- it's the product or service which someone is *requesting* to book. It's not a product, it's a *request* to book a product or service.

Reply:
https://github.com/KarmaComputing/collect-deposit-online/pull/17/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R8